### ISSUE TITLE
calendar icon covers date field

### ISSUE THREAD

**Issue by metas-lc** [metasfresh/metasfresh-webui-frontend#1428]
### Is this a bug or feature request?
fr
### What is the current behavior?
when editing a date in included tab gridview, the icon for calendar covers the date. If you want to see calendar, you have to press that icon
#### Which are the steps to reproduce?
1. open product window, select one e.g. https://w101.metasfresh.com:8443/window/140/2005572
2. go to cu-tu allocation tab
3. double click to edit valid from 
![image](https://user-images.githubusercontent.com/16001666/33765563-0c197cf8-dc22-11e7-9a5c-e7379a2a6e0e.png)

=> the icon covers the date

### What is the expected or desired behavior?
should not see that icon, and the calendar shall appear by default

**Comment by ottosichert** [comment]
Probably introduced by https://github.com/metasfresh/metasfresh-webui-frontend/pull/1398

**Comment by wiadev** [comment]
@metas-lc can you check this issue if it's still valid?

**Comment by metas-lc** [comment]
@wiadev i checked. issue is still valid

**Comment by wiadev** [comment]
@teosarca 
Can you update the size of Date to "M" on back-end?

![screenshot at jan 11 12-20-45](https://user-images.githubusercontent.com/18234060/34840871-fa5e3494-f6cb-11e7-8702-881e6f1c4e92.png)

**Comment by teosarca** [comment]
@metas-mk  wdyt?

**Comment by metas-mk** [comment]
@wiadev 
Just checked locally. Seems as if the Calendar fields are now not aligned properly anymore.
See Sales order window window/143/1000311 as example. Pls also check in others.
![image](https://user-images.githubusercontent.com/13365687/34938588-29764d28-f9e9-11e7-981b-71f7eba92730.png)

**Comment by metas-mk** [comment]
Restarted Server and checked again. Looks like this now
![image](https://user-images.githubusercontent.com/13365687/34938857-0cd18dee-f9ea-11e7-8aa3-0dc94d9b29c2.png)
a) Icons are moved into the end of content (should be end of field)
b) In case of date field (not datetime) the field length is reduced and bottom line is not expanded
c) the calendar icons are smaller than before in main view. Please use the larger size in main view as before.

**Comment by metas-lc** [comment]
IT
1. checked again "valid from" in window 140
=> the icon has a white background now, so it's not over the date, even if now part of date is hidden OK
=> the calendar appears same as icon on double click, but it's way higher. I think because it must fit on the page (have to check if it's wanted)
 
2. checked other date fields => are aligned properly and the icons are at the end of the field OK

3. length of date fields are like on date+time fields OK

4. icons are still small NOK
OLD:
![image](https://user-images.githubusercontent.com/16001666/35219119-f2229be2-ff79-11e7-9970-ee7b4ba27774.png)

NEW:
![image](https://user-images.githubusercontent.com/16001666/35219166-2bfaaa3a-ff7a-11e7-9960-3ad54f7e5ff2.png)

**Comment by siemiatj** [comment]
Is it only the size of the calendar icon that's left here 
![screen shot 2018-04-03 at 9 25 14 pm](https://user-images.githubusercontent.com/513460/38271959-2aebbd00-3788-11e8-95d3-3cdc7cc01cc6.png)

or do we want to somehow figure out a better placement of the calendar in table ?

![screen shot 2018-04-03 at 9 44 58 pm](https://user-images.githubusercontent.com/513460/38272026-56a9335a-3788-11e8-9af2-69577b0c1285.png)

**Comment by siemiatj** [comment]
ping @metas-lc @metas-mk ^^

**Comment by metas-lc** [comment]
hi. i think it would be nice to have it right under the field. @metas-mk wdyt? 
i added some new lines here to see how it looks 
![image](https://user-images.githubusercontent.com/16001666/40052389-733f6f64-5846-11e8-87ed-a0fab78590eb.png)

**Comment by teosarca** [comment]
> 3. till cannot see save action. Cannot complete temp pricing cond order and pricing conditions action missing in purchase order NOK

Yup, because that BPartner does not have discount schema defined.
How can you find that out now (i know, not obvious):
* take a look at the quickActions endpoint. Right click on it and open it in a new tab:
![image](https://user-images.githubusercontent.com/1244701/40101547-6c945a58-58f0-11e8-89e7-b1e30a95435d.png)

* edit the link and append `&all=true`. You will get all actions. Those disabled will have `disabled: true` but you also get the reason.
![image](https://user-images.githubusercontent.com/1244701/40101605-a95625a2-58f0-11e8-835c-5a4f27af1223.png)


Talked with @metas-mk  and we agreed it's acceptable for now.
In future the admin/power user could have some `right click -> inspect` option somewhere, will get all the actions will see why those are not available.

**Comment by siemiatj** [comment]
I think this might be problematic, as in case there's no room for the calendar we'd have to programatically make the window bigger.
